test/runtests.jl

```diff
@@ -258,18 +259,18 @@ end
 end
 
 @testset "full-text search" begin
-    function linear_search(query)
+    global function linear_search(query)
         locs = Int[]
         loc = 0
-        while (loc = searchindex(text, Vector{UInt8}(query), loc + 1)) > 0
+        while (loc = first(something(findnext(query, textstr, loc + 1), 0:-1))) > 0
             push!(locs, loc)
         end
         return locs
     end
 
-    text = open(read, joinpath(dirname(@__FILE__), "lorem_ipsum.txt"))
+    text = read(joinpath(dirname(@__FILE__), "lorem_ipsum.txt"))
+    textstr = String(copy(text))
```

Comment:
This doesn't look like it gets used... maybe meant [for here](https://github.com/BioJulia/FMIndexes.jl/pull/9/files#diff-fce720c43af3c52c862fd7451c7374b8R288)?

Reply:
lol yeah this isn't doing anything, I think I was testing something and forgot to delete this.

Reply:
`textstr` is used for the linear_search. I'm going to pass it into the local linear_search as an arg though and remove the global.